pkg/suggestion/cma_service.py

```diff
@@ -0,0 +1,213 @@
+import json
+
+import grpc
+import numpy as np
+
+from pkg.api.python import api_pb2
+from pkg.api.python import api_pb2_grpc
+from pkg.suggestion.cma.src.algorithm_manager import AlgorithmManager
+from pkg.suggestion.cma.src.cma_algorithm import CMAES
+from pkg.suggestion.types import MANAGER
+
+_ONE_DAY_IN_SECONDS = 60 * 60 * 24
+
+
+class CMAService(api_pb2_grpc.SuggestionServicer):
+    def __init__(self):
+        channel = grpc.insecure_channel(MANAGER)
+        self.stub = api_pb2_grpc.ManagerStub(channel)
+
+    def GetSuggestions(self, request, context):
```

Comment:
What happens when you call GetSuggestion before previous Trials are completed.

Reply:
So if the previous trials are not complete, I assume that when I call `GetMetrics`, no `metircs_log_sets` will be returned. Then I will return an grpc error saying that all the trials in previous population should be evaluated.
like this:
https://github.com/libbyandhelen/hp-tuning-1/blob/b98dccb5cbc70e85004c848cc7cd7ea403a39577/pkg/suggestion/cma_service.py#L95Is my assumption correct or not?

Reply:
@YujiOshima 
Then can I check the status of the worker after calling `GetWorkers` method to see whether the worker is completed?
From your code, I see that after the client calls `RunTrial`, it will create a worker in db (Pending state) and spawn a worker (Running state), but where is code that updates the worker state to `completed` after the evaluation?

Reply:
@libbyandhelen When you call `GetMetrics`, the status of workers is updated.
So you can get the latest status to call `GetWorker` after `GetMetrics`.
If you think it troubles, we can add `status` item to `GetMetricsReply`.
WDYT?

Reply:
@YujiOshima 
Thanks! The additional `status` item may be more natural for me. Could you please add it?